NEXTCLOUD.cmake

```diff
@@ -15,6 +15,30 @@ set( APPLICATION_VIRTUALFILE_SUFFIX "nextcloud" CACHE STRING "Virtual file suffi
 set( LINUX_PACKAGE_SHORTNAME "nextcloud" )
 set( LINUX_APPLICATION_ID "${APPLICATION_REV_DOMAIN}.${LINUX_PACKAGE_SHORTNAME}")
 
+option(DEV_BUILD "If enabled, name variables will be appended with safe variations of DEV_SUFFIX in order to reduce conflicts." OFF)
+
+if (NOT DEV_SUFFIX)
+    set(DEV_SUFFIX "(Test Build)"
```

Comment:
Please use just "Dev" as suffix. Then you don't need to do all this string manipulation that could cause errors and as I said I see no value in using parenthesis and the word Build. `APPLICATION_NAME` appears in the interface and in my opinion "Nextcloud Daily" sounds and looks much better than "Nextcloud (Daily Build)". It looks less cryptic. Please consider also to use the word Dev instead of Test. After all we name the option `DEV_BUILD` and not `TEST_BUILD`.

Reply:
I could change the default value of `DEV_SUFFIX` to "Dev" because, yes, that makes sense. I would prefer to leave the string manipulation, though, since, no, the three specific things I included will not cause errors. If you dislike string manipulation, I will direct your attention to these lines, which I am directly aping:
https://github.com/nextcloud/desktop/blob/db0f1e245df27e5673df0931da0e4e0f56e1c0b6/CMakeLists.txt#L33-L36

Reply:
I went ahead and changed the default to match the name of the variable.